webui/react/src/routes/index.ts

```diff
@@ -113,12 +113,12 @@ export const defaultAppRoute = dashboardRoute;
 export const appRoutes: RouteConfig[] = [
   dashboardRoute,
   experimentDetailsRoute,
+  trialLogsRoute,
```

Comment:
👍

Reply:
shouldn't be needed anymore. I'll take it out